workloads/run_workloads.yaml

```
@@ -436,10 +436,17 @@
         own_ip: "{{application_host_ip}}"
         communication_port: 0
       environment: "{{workload_version.env | default({}) }}"
+      block:
+      - name: Mesos | Create stress_ng
+        shell: "{{create_job_command}}"
+        loop: "{{instances_version_iterator}}"
+        when: orchestrator == "Mesos"
+      - name: Kubernetes | Create stress_ng
```

It shouldn't be this way?
```suggestion
      - name: Kubernetes | Create tensorflow-benchmark/train workload
```